### DISCUSSION TITLE
What impact does backup:run have on the server?

### DISCUSSION THREAD

**Discussion by sebsobseb** [spatie/laravel-backup#1521] [Q&A]
We have some users complaining about the website being down for 30s - 1min at the same time we have scheduled to run `backup:run --only-db`. What is the recommended way to run a backup without affecting the server performance?

**Comment by PaolaRuby** [comment]
make the backup at times that do not have visitors, at dawn for example, maybe the db is too big

**Reply by sebsobseb** [reply]
Our site (game) is being used word-wide across multiple timezones, so unfortunately we don't really have a time where there are no visitors. The backup is around 370Mb

**Reply by PaolaRuby** [reply]
[Fix website down on backup](https://www.google.com/search?q=website+down+when+db+backup&oq=website+down+when+db+backup)
or make a PR for better performance

**Reply by sebsobseb** [reply]
Would running the command in a Job help?

**Reply by PaolaRuby** [reply]
Did you at least read what the code does?
Make a PR if you find a way
